### PR TITLE
Update billing dates typing

### PR DESCRIPTION
**Pull request by DmitriyNoa**

### CODE REVIEW

File: src/UserService.ts [hunk 1]
```diff
@@ -47,10 +47,22 @@ export type SubscriptionInfo = {
   id?: string | null;
   planName?: string | null;
   planCode?: string | null;
+  /**
+   * Current payment period start date string
+   */
   currentPeriodStartedAt?: Date | null;
```
[the remark this answers is not in the document]
Comment (DmitriyNoa) [reply src/UserService.ts]: Sorry, forgot to update these 2 as well

[the remark this answers is not in the document]
Comment (DmitriyNoa) [reply src/UserService.ts]: Now done

**Review by chenhunghan — COMMENTED** [review]
Maybe it's too late to say, but why do need `null`? Can't them be just `string | undefined` like
```
currentPeriodStartedAt?: string;
```

**Comment by DmitriyNoa** [comment]
@chenhunghan the types from Recurly are s disaster. They have it as 
`currentPeriodStartedAt?: string | null | undefined;` internally. We decided to return string or null for simplicity

**Review by chenhunghan — APPROVED** [review]

**Pull request merged**